core/storage/suggestion/gae_models.py

```diff
@@ -2214,7 +2214,7 @@ def fetch_page(
         page_size: int,
         offset: int,
         language_code: str,
-        sort_by: Optional[SortChoices],
+        sort_by: Optional[str],
```

Comment:
Why change it from enum to str?

Reply:
Actually while trying to address https://github.com/oppia/oppia/pull/18576#discussion_r1249593166 this. I changed it from enum to str but we shouldn't be loose on type checking therefore added it back again.